### PR TITLE
Task scheduler

### PR DESCRIPTION
**Pull request by Keccak1**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+cpp_space_before_inheritance_colon=true
+cpp_space_before_constructor_colon=true
+cpp_space_remove_before_semicolon=true
+cpp_space_after_semicolon=false
+cpp_space_remove_around_unary_operator=true
+cpp_space_around_binary_operator=insert
+cpp_space_around_assignment_operator=insert
+cpp_space_pointer_reference_alignment=left
+cpp_space_around_ternary_operator=insert
+cpp_wrap_preserve_blocks=one_liners
```
Comment (kamilmachon) [inline .editorconfig:59]: shouldn't we include this file in gitignore?

File: mk_robotics/ws_manager/WebSocketManager.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            {
+                std::for_each(_clients.begin(), _clients.end(), f);
+            }
+            
+            void notifyAll(const std::string &msg);
+            void add(const drogon::WebSocketConnectionPtr &);
+            
+
+        private:
+            std::vector<drogon::WebSocketConnectionPtr> _clients;
```
Comment (kamilmachon) [inline mk_robotics/ws_manager/WebSocketManager.h]: Are we trying to unify code conventions in motor_driver and drogon repository?  I started using PascalCase in methods and underscore after class variable name

**Review by kamilmachon — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (Keccak1) [reply .editorconfig:59]: No, I use the VS on windows to debug, and that file allows me to keep the same style in both IDEs

File: mk_robotics/ws_manager/WebSocketManager.h (hunk 2 above)
Comment (Keccak1) [reply mk_robotics/ws_manager/WebSocketManager.h]: Good point

**Pull request merged**